src/fiskaltrust.Middleware.Interface.Client.Http/POS/HttpPos.cs

```diff
@@ -21,6 +21,8 @@ internal class HttpPos : IPOS
         private delegate ifPOS.v0.ReceiptResponse AsyncSignCaller(ifPOS.v0.ReceiptRequest request);
         private delegate Stream AsyncJournalCaller(long ftJournalType, long from, long to);
 
+        private string V0VersionUrl => _options.UseUnversionedLegacyUrls ? "" : "v0/";
```

Comment:
Private properties are usually not a clear pattern - we could make this a private field `_v0VersionUrl`, and set it in the constructor